src/hooks/useTransition.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react'
 import { usePrevious } from './usePrevious'
 
-const EXITED = 'exited'
-const ENTERING = 'entering'
-const ENTERED = 'entered'
-const EXITING = 'exiting'
+export const EXITED = 'exited'
+export const ENTERING = 'entering'
+export const ENTERED = 'entered'
+export const EXITING = 'exiting'
```

Comment:
Check the rest of the codebase, we can use these constants there as well (f.e. dialog, content transition)

Reply:
I'll do this in a separate PR, so I can test everything and make sure it still works 😅